projects/storefrontapp/src/app/spartacus/features/segment-refs/segment-refs-feature.module.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { NgModule } from '@angular/core';
+import { SegmentRefsModule } from '@spartacus/segment-refs';
+
+@NgModule({
+  imports: [SegmentRefsModule],
```

Comment:
According to above comments, let's use `SegmentRefsRootModule` here

Reply:
done